scheduler/api/grpc/doc.go

```diff
@@ -0,0 +1,4 @@
+/*
```

Comment:
Until we populate these we shouldn't over pattern, let's get rid of all the doc only packages.